src/components/NavBar/NavBar.html

```diff
@@ -0,0 +1,13 @@
+<html>
+<head>
+    <link rel="stylesheet" href="NavBar.css">
+</head>
+<body>
+
+<div class="navbar">
+    <a href="#">Logout</a>
+    <a href="#">Edit Profile</a>
+</div>
+
+</body>
+</html>
```

Comment:
Can you change this to a .jsx file? Check LoginPage.jsx for inspiration. Sorry I didn't specify this earlier, but it should be a quick change.